File: malevich-server/src/main/java/io/malevich/server/transfer/GalleryDto.java
package io.malevich.server.transfer;

import com.yinyang.core.server.transfer.FileDto;
import lombok.Getter;
import lombok.Setter;

import java.util.Map;

@Getter
@Setter
public class GalleryDto extends ParticipantDto {

    private OrganizationDto organization;

    private FileDto thumbnail;

    private FileDto image;

    private Map<String, String> titleMl;

    private Map<String, String> descriptionMl;
}

File: malevich-web/src/app/_transfer/galleryDto.ts
import {OrganizationDto} from "./organizationDto";
import {ParticipantDto} from "./participantDto";
import {FileDto} from "../../../node_modules/yinyang-core";

export class GalleryDto extends ParticipantDto{
  organization: OrganizationDto;
  thumbnail: FileDto;
  image: FileDto;
  descriptionMl: Map<string, string>;
}
